packages/panels/src/Commands/MakeThemeCommand.php

```diff
@@ -108,7 +115,7 @@ public function handle(): int
         $this->components->bulletList([
             "First, add a new item to the `input` array of `vite.config.js`: `resources/css/filament/{$panelId}/theme.css`.",
             "Next, register the theme in the {$panelId} panel provider using `->viteTheme('resources/css/filament/{$panelId}/theme.css')`",
-            'Finally, run `npm run build` to compile the theme.',
+            "Finally, run `{$pm} run build` to compile the theme.",
```

Comment:
I've never tried with `bun` but `bun build` is the command right, not `bun run build`? https://bun.sh/docs/bundler

Reply:
tried both build and run build both works, same with yarn works yarn run build and yarn build